web/src/RootSSHKey.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import { FileUpload } from "@patternfly/react-core";
+
+export default function RootSSHKey(props) {
+  const [loading, setLoading] = useState(false);
+
+  return (
+    <FileUpload
+      id="SSHKey"
+      type="text"
+      value={props.value}
+      filenamePlaceholder="Drag and drop an SSH public key or upload one"
+      onDataChange={value => {
+        props.valueChanged(value);
+      }}
+      onTextChange={value => {
+        props.valueChanged(value);
+      }}
+      onReadStarted={() => setLoading(true)}
+      onReadFinished={() => setLoading(false)}
+      onClearClick={() => props.valueChanged("")}
+      isLoading={loading}
+      browseButtonText="Upload"
```

Comment:
NP: Upload or Search. Not sure wich is better/common.

Reply:
Same here. I keep example from pattern fly as it was for me obviously bad.

Reply:
I see. And it is true that the action is a "Browse & Upload"... maybe the Upload label is not that bad. No strong opinion.